modules/API/external/external_api.js

```diff
@@ -81,6 +81,26 @@ function changeParticipantNumber(APIInstance, number) {
 }
 
 /**
+ * Generates array with URL params based on the passed config object that will
+ * be used for the Jitsi Meet URL generation.
+ *
+ * @param config {object} the config object.
+ * @returns {Array<string>} the array with URL param strings.
+ */
+function configToURLParamsArray(config) {
```

Comment:
Do you mean that there isn't a empty line between `const params = [];` and the rest of the code in the function? I've just fixed it!